src/model.js

```diff
@@ -2725,11 +2726,11 @@ class Model {
         }
       }
 
-      if (options.ignoreDuplicates && ['mssql', 'db2'].includes(dialect)) {
+      if (options.ignoreDuplicates && ['mssql', 'db2', 'ibmi'].includes(dialect)) {
```

Comment:
maybe out of scope for this PR but I'd prefer if we had a flag on Dialect.supports instead of hardcoding dialects here